SpatialGDK/Source/SpatialGDK/Private/Interop/Connection/SpatialConnectionManager.cpp

```diff
@@ -79,20 +79,19 @@ struct ConfigureConnection
 		Params.network.modular_kcp.upstream_heartbeat = &HeartbeatParams;
 #endif
 
-		if (!bConnectAsClient && GetDefault<USpatialGDKSettings>()->bUseSecureServerConnection)
-		{
-			Params.network.modular_kcp.security_type = WORKER_NETWORK_SECURITY_TYPE_TLS;
-			Params.network.modular_tcp.security_type = WORKER_NETWORK_SECURITY_TYPE_TLS;
-		}
-		else if (bConnectAsClient && GetDefault<USpatialGDKSettings>()->bUseSecureClientConnection)
+		// Use insecure connections default.
+		Params.network.modular_kcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
+		Params.network.modular_tcp.security_type = WORKER_NETWORK_SECURITY_TYPE_INSECURE;
+
+		// Override the security type to be secure only if the user has requested it and we are not using an editor build.
+		if ((!bConnectAsClient && GetDefault<USpatialGDKSettings>()->bUseSecureServerConnection) || (bConnectAsClient && GetDefault<USpatialGDKSettings>()->bUseSecureClientConnection))
 		{
+#if WITH_EDITOR
+			UE_LOG(LogSpatialWorkerConnection, Warning, TEXT("Secure connection requested but this is not supported in Editor builds. Connection will be insecure."));
```

Comment:
Why can't we support secure connections in the editor?

Reply:
New runtime explicitly has this disabled.

Reply:
Can you elaborate Josh? I could technically run an editor binary and connect to a cloud deployment. Surely I could have secure connection on then if I wished?

Reply:
Deemed that this is unneeded from a product perspective. Additionally if this really is required, it's simple to workaround for a known customer.

Reply:
This looks like an arbitrary limitation. Even if it's unneeded from a product perspective, I see no reason to shave off features that could work (unless making the decision on if we are connecting to a local deployment is complicated).
I'd rather have something that "just works" instead of having to do a support ticket roundtrip.

Reply:
Interesting, how do you propose we detect that you're forming a local connection versus a cloud connection then to solve this for all potential cases. If we decide to support this.

Reply:
Chatted with @ernopp about this and from a product perspective we don't require the secure connection for editor -> cloud. As long as it's well documented this only works for non editor builds then we're fine.

Reply:
Thanks Josh!